GDJS/Runtime/gd.d.ts

```diff
@@ -0,0 +1,68 @@
+// Type definitions for gd.js
+// Project: GDJS Platform of GDevelop
```

Comment:
nitpicking: GDJS Platform of GDevelop means "GDevelop JavaScript Platform of GDevelop" ;)
Prefer "GDevelop JavaScript Runtime" or "GDevelop JavaScript Platform".

Reply:
I know that, but What I meant is the GDJS component of the GDevelop project